templates/index.html

```diff
@@ -92,7 +92,7 @@ <h4 class="p-testimonial__title">“A major software discovery tool”</h4>
 <div class="p-strip is-deep is-bordered">
   <div class="row">
     <div class="col-6">
-      <h3>One build for all Linux and IoT</h3>
+      <h3 class="p-heading--two">One build for all Linux and IoT</h3>
```

Comment:
@Lukewh As far as I can see `h3`s get `p-heading-two` class here.

Categories headings on Store page also use `h3`. Shouldn't they get the class as well?

Reply:
@bartaz No, the categories on /store should get .p-heading--four.